From cb258dcdd39f8fc6b384e165544b4c230633c647 Mon Sep 17 00:00:00 2001
From: Huiquan Zhong <huiquan.zhong@intel.com>
Date: Thu, 7 Aug 2014 10:00:09 +0800
Subject: [PATCH] dw_i2c: add Intel Cherrytrail I2C ACPI ID

Add Intel Cherrytrail I2C ACPI ID(808622C1).

Change-Id: Ide1eb9616b867761a2757bdf83c23e11f4c19acd
Signed-off-by: Huiquan Zhong <huiquan.zhong@intel.com>
(cherry picked from commit ac927c1ee17d96bf4860c8e3d463646630a91b22)

Signed-off-by: Brett T. Warden <brett.t.warden@intel.com>
---
 drivers/i2c/busses/i2c-designware-platdrv.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/i2c/busses/i2c-designware-platdrv.c b/drivers/i2c/busses/i2c-designware-platdrv.c
index 61b40f5..b62df28 100644
--- a/drivers/i2c/busses/i2c-designware-platdrv.c
+++ b/drivers/i2c/busses/i2c-designware-platdrv.c
@@ -106,6 +106,7 @@ static const struct acpi_device_id dw_i2c_acpi_match[] = {
 	{ "INT3432", 0 },
 	{ "INT3433", 0 },
 	{ "80860F41", 0 },
+	{ "808622C1", 0 },
 	{ }
 };
 MODULE_DEVICE_TABLE(acpi, dw_i2c_acpi_match);
-- 
1.7.9.5

